### DISCUSSION TITLE
How to voxelize Nxd tensor

### DISCUSSION THREAD

**Discussion by digital-idiot** [pyg-team/pytorch_geometric#2459] [Q&A]
I am working on voxel based approaches. I wonder if I can adequately voxelize `Nxd` tensor. Here `d >3`, i.e., `x, y, z, ...<features>...`

**Comment by rusty1s** [comment]
PyG is a library for accelerating deep learning on irregularly structured data, and so it isn't a great fit for voxel-based approaches.
